cirq/ops/common_channels.py

```diff
@@ -187,3 +189,528 @@ def depolarize(p: float) -> DepolarizingChannel:
         ValueError if p is not a valid probability.
     """
     return DepolarizingChannel(p)
+
+
+class GeneralizedAmplitudeDampingChannel(raw_types.Gate):
+    """ The generalized amplitude damping channel."""
```

Comment:
Don't just repeat the name, define what the operation actually does (i.e. move most of the content from `__init__` to here).

Reply:
fixed

Reply:
It doesn't look fixed...? Did you push?

Reply:
fixed